tf2onnx/tfonnx.py

```diff
@@ -1877,6 +1922,9 @@ def zeroslike_op(ctx, node, name, args):
     "Asinh": (direct_op, []),
     "Acosh": (direct_op, []),
     "Atanh": (direct_op, []),
+    "ResizeBilinear": (upsample_op9, ["Upsample", "linear"]),
+    "ResizeNearestNeighbor": (upsample_op9, ["Upsample", "nearest"]),
+    "Slice": (dynamic_slice_op, []),
```

Comment:
if it can't work later then the failure of test case will notify us
why bother to add a comment since if it continues to working well then we may forget to remove the comment added.